### PR TITLE
1.1.2

### PR DESCRIPTION
**Pull request by DenBond7**
- [x] fix tests

### CODE REVIEW

**Comment by tomholub** [comment]
It's looking better. Here are some Codacy recommendations. I gave you permissions to view it now (I hope).

image

image

**Comment by tomholub** [comment]
Remember to look at https://github.com/FlowCrypt/flowcrypt-android/pull/956#issuecomment-699493117 - it's an easy fix and then you'll see clearer which commits are green and which red.

**Comment by tomholub** [comment]
About how many tests are currently running in CI, and about how many are not?

**Comment by DenBond7** [comment]
@tomholub 
We have 224 tests at all. All of them already work locally


> currently running in CI

189 tests

**Comment by DenBond7** [comment]
image

**Comment by tomholub** [comment]
Should probably rename them, "Tests without mailserver" is no longer current :)

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/email/protocol/PropertiesHelper.kt [hunk 1]
```diff
@@ -88,6 +89,10 @@ class PropertiesHelper {
         prop[JavaEmailConstants.PROPERTY_NAME_MAIL_SMTP_STARTTLS_ENABLE] = it.smtpOpt === SecurityType.Option.STARTLS
         prop[JavaEmailConstants.PROPERTY_NAME_MAIL_SMTP_SSL_CHECK_SERVER_IDENTITY] = it.smtpOpt === SecurityType.Option.SSL_TLS
 
+        prop[JavaEmailConstants.PROPERTY_NAME_MAIL_IMAPS_SSL_TRUST] = Constants.HOST_MACHINE_IP
+        prop[JavaEmailConstants.PROPERTY_NAME_MAIL_IMAP_SSL_TRUST] = Constants.HOST_MACHINE_IP
+        prop[JavaEmailConstants.PROPERTY_NAME_MAIL_SMTP_SSL_TRUST] = Constants.HOST_MACHINE_IP
```
Comment (tomholub) [inline FlowCrypt/src/main/java/com/flowcrypt/email/api/email/protocol/PropertiesHelper.kt:94]: This looks definitely problematic to have in app code

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/email/protocol/PropertiesHelper.kt [hunk 2]
```diff
@@ -122,6 +127,10 @@ class PropertiesHelper {
         prop[JavaEmailConstants.PROPERTY_NAME_MAIL_SMTP_STARTTLS_ENABLE] = it.smtpOpt() === SecurityType.Option.STARTLS
         prop[JavaEmailConstants.PROPERTY_NAME_MAIL_SMTP_SSL_CHECK_SERVER_IDENTITY] = it.smtpOpt() === SecurityType.Option.SSL_TLS
 
+        prop[JavaEmailConstants.PROPERTY_NAME_MAIL_IMAPS_SSL_TRUST] = Constants.HOST_MACHINE_IP
```
Comment (tomholub) [inline FlowCrypt/src/main/java/com/flowcrypt/email/api/email/protocol/PropertiesHelper.kt:130]: And this

**Pull request merged**